core/src/main/java/io/servicecomb/core/Invocation.java

```diff
@@ -53,15 +53,14 @@
   // 同步模式：避免应答在网络线程中处理解码等等业务级逻辑
   private Executor responseExecutor;
 
-  //start,end of queue and opertion time after queue for operation level metrics.
-  private Object metricsData;
+  private long startProcessingTime;
 
-  public Object getMetricsData() {
-    return metricsData;
+  public long getStartProcessingTime() {
```

Comment:
Suggestion: Better to add java doc for newly added methods. Applicable for all newly added methods.

Reply:
Comment added